### PR TITLE
Changing ps1 script in demo 6.

### PR DESCRIPTION
**Pull request by cesarfong**
# Description

1) Remove syntaxis errors. Adding { } to 10 variables in the script.

 2) Using helm to install keda.

## Issue reference

We strive to have all PRs being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: # it refers to here if we use the command func kubernetes https://github.com/Azure/azure-functions-core-tools/issues/1883

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] The sample code compiles correctly
* [x] You've tested new builds of the sample if you changed sample code
* [x] You've updated the sample's README if necessary

### CODE REVIEW

File: 6.functions-and-keda/setup.ps1 [hunk 1]
```diff
@@ -30,6 +30,7 @@ az group create -n $groupName -l $location
 Write-Host
 Write-Host "Creating AKS cluster $clusterName..."
 az aks create -g $groupName -n $clusterName --generate-ssh-keys
+# The next line failed the first 2 times.
```
Comment (pruthvidhodda) [inline 6.functions-and-keda/setup.ps1:33]: Can we remove this line?

Comment (cesarfong) [reply 6.functions-and-keda/setup.ps1:33]: ok i will remove

**Comment by cesarfong** [comment]
I have eliminate the comment. 

Please let me know.

**Comment by pruthvidhodda** [comment]
@cesarfong can you please update your branch with the base branch so that I can merge this PR.

**Comment by cesarfong** [comment]
Updated

**Comment by pruthvidhodda** [comment]
@cesarfong thanks for your contribution! :)

**Pull request merged**